templates/adherent/_profile_menu.html.twig

```diff
@@ -10,8 +10,8 @@
             <a href="{{ path('app_user_edit') }}">Mes informations personnelles</a>
         </li>
 
-        <li class="{{ route == 'app_user_profile_donation' ? 'active' }}">
-            <a href="{{ path('app_user_profile_donation') }}">Mes dons</a>
+        <li class="{{ route == 'app_user_profile_donation' ? 'active' }}" >
+            <a href="{{ path('app_user_profile_donation') }}" id="my_donations">Mes dons</a>
```

Comment:
Because in the test he say that "Me dons" is hidden because this label is displayed in the user hidden menu. He stop his search at the first element he find and if he is hidden he say that he can't click on it.

Reply:
What about clicking on the menu before clicking on this link ?

Reply:
It wasn't the purpose of this test :s 
```behat

    When I click the "donation_check_label" element
    And I click the "donation_check_nationality_label" element
    And I press "Je donne"
    Then I should be on "/don/coordonnees?montant=50&abonnement=1"
    And I should see "Vous faites déjà un don mensuel à La République En Marche ! Vous pouvez vous rendre sur votre profil pour l’annuler ou faire un nouveau don unique."

    When I follow "vous rendre sur votre profil"
    Then I should be on "/parametres/mon-compte"

    When I follow "my_donations"
    And I follow "Mettre fin à mon don mensuel"
    And I press "Oui"
    Then I should see "Votre don mensuel a bien été annulé. Vous recevrez bientôt un mail de confirmation."
```

Reply:
No but it is a requirement and if we can avoid adding useless IDs it is great.

Reply:
Of course